src/fidesops/models/fidesops_user.py

```diff
@@ -48,3 +57,13 @@ def credentials_valid(self, password: str) -> bool:
         )
 
         return provided_password_hash == self.hashed_password
+
+    def update_password(self, db: Session, new_password: str) -> None:
+        """Updates the user's password to the specified value.
+        No validations are performed on the old/existing password within this function."""
+
+        hashed_password, salt = FidesopsUser.hash_password(new_password)
+        self.hashed_password = hashed_password
+        self.salt = salt
+        self.password_reset_at = datetime.utcnow()
+        self.save(db)
```

Comment:
Is there another ticket that adds an API endpoint to update the password/validate that the existing password is correct before doing so? If not, can you create one?

Reply:
i took a shot with https://github.com/ethyca/fidesops/issues/404 - happy to discuss/adjust as needed. thanks!

Reply:
looks good @adamsachs thanks